llvm/test/tools/sycl-post-link/exclude_external_functions.ll

```diff
@@ -0,0 +1,67 @@
+; This test checks that when device code splitting is done for run-time linking,
+; dependencies to SYCL External functions do not cause the SYCL External function
+; to be added to a device image.
+
+; RUN: sycl-post-link -symbols -exclude-external-functions -split=kernel -S < %s -o %t.table
+
+
+; RUN: FileCheck %s -input-file=%t_0.sym --check-prefixes CHECK-SYM-0
+; RUN: FileCheck %s -input-file=%t_1.sym --check-prefixes CHECK-SYM-1
```

Comment:
it seems we have two LIT tests but they're exactly the same minus a comment, do we want to delete one of them?

Reply:
Deleted poorly named file.